source/extensions/filters/network/sni_cluster/config.cc

```diff
@@ -0,0 +1,41 @@
+#include "extensions/filters/network/sni_cluster/config.h"
+
+#include "envoy/registry/registry.h"
+#include "envoy/server/filter_config.h"
+
+#include "extensions/filters/network/sni_cluster/sni_cluster.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace SniCluster {
+
+Network::FilterFactoryCb
+SniClusterNetworkFilterConfigFactory::createFilterFactory(const Json::Object&,
+                                                          Server::Configuration::FactoryContext&) {
+  // Only used in v1 filters.
+  NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
```

Comment:
Since v1 will be completely removed soon, should we mark this method non-PURE and in NamedNetworkFilterConfigFactory and make it `NOT_IMPLEMENTED`?

Reply:
I was waiting for the 1.8 cycle. after that, we could remove these methods entirely from everywhere.